build.gradle

```diff
@@ -198,6 +195,7 @@ project.ext.getDockerImageNameShortCommit = {
 }
 
 docker {
+    dependsOn copyBootJar
```

Comment:
FYI: this makes building the docker image directly with the command `docker image build ...` more complicated.

Reply:
I don't understand your point. I mean it is down right impossible in the other repositories due to a templated Dockerfile.
The only and right way to build docker images is to do `./gradlew buildImage` or `./gradlew docker`.

Reply:
It's not that I don't understand your point, it's that it is a problem everywhere and should be addressed in a broader story on how we would like to be able to build docker images for all applications.

Reply:
Fair enough yes :+1: . It was just a heads-up.